docs/es/week12/12-2.md

```diff
@@ -0,0 +1,600 @@
+
+- ¡Muchos objetivos diferentes de preentrenamiento funcionan bien!
+
+- Crucial para modelar interacciones profundas y bidireccionales entre palabras
+
+- Grandes beneficios de agrandar el preentrenamiento, aún sin límites claros
+
+<!--Most of the models discussed above are engineered towards solving the text classification problem. However, in order to solve text generation problem, where we generate output sequentially much like the `seq2seq` model, we need a slightly different approach to pre training.-->
+
+La mayoría de los modelos discutidos anteriormente están diseñados para resolver el problema de clasificación de texto. Sin embargo, para resolver el problema de la generación de texto, donde generamos resultados secuencialmente como el modelo `seq2seq`, necesitamos un enfoque ligeramente diferente al preentrenamiento.
+
+<!--#### Pre-training for Conditional Generation: BART and T5-->
+
+#### Entrenamiento previo para generación condicional: BART y T5
+
+<!--BART: pre-training `seq2seq` models by de-noising text-->
+
+BART: preentrenamiento de modelos `seq2seq` mediante la eliminación de ruido en el texto
+
+<!--In BART, for pretraining we take a sentence and corrupt it by masking tokens randomly. Instead of predicting the masking tokens (like in the BERT objective), we feed the entire corrupted sequence and try to predict the entire correct sequence.-->
+
+En BART, para el entrenamiento previo, tomamos una oración y la corrompemos enmascarando tokens al azar. En lugar de predecir los tokens de enmascaramiento (como en el objetivo de BERT), alimentamos la secuencia corrupta completa e intentamos predecir la secuencia correcta completa.
+
+<!--This `seq2seq` pretraining approach give us flexibility in designing our corruption schemes. We can shuffle the sentences, remove phrases, introduce new phrases, etc.-->
+
+Este enfoque de preentrenamiento `seq2seq` nos da flexibilidad para diseñar nuestros esquemas de corrupción. Podemos barajar las oraciones, eliminar frases, introducir nuevas frases, etc.
+
+
+<!--BART was able to match RoBERTa on SQUAD and GLUE tasks. However, it was the new SOTA on summarization, dialogue and abstractive QA datasets. These results reinforce our motivation for BART, being better at text generation tasks than BERT/RoBERTa.-->
+
+BART pudo igualar a RoBERTa en las tareas SQUAD y GLUE. Sin embargo, fue el nuevo estado del arte sobre resumenes, diálogos y conjuntos de datos de control de calidad abstractos. Estos resultados refuerzan nuestra motivación por BART, siendo mejores en tareas de generación de texto que BERT/RoBERTa.
+
+<!--### Some open questions in NLP
+
+- How should we integrate world knowledge
+- How do we model long documents?  (BERT-based models typically use 512 tokens)
+- How do we best do multi-task learning?
+- Can we fine-tune with less data?
+- Are these models really understanding language?-->
+
+### Algunas preguntas abiertas en NLP
+
+- ¿Cómo deberíamos integrar el conocimiento mundial?
```

Comment:
In the previous PR, I suggested using "conocimiento del mundo real" instead of "conocimiento mundial", as that's what I've seen most commonly used in text for LATAM, but I'm not sure if the same applies in other Spanish-speaking regions of the world (particularly Spain)